src/views/workflow-history-v2/hooks/use-workflow-history-fetcher.ts

```diff
@@ -0,0 +1,100 @@
+import { useCallback, useEffect, useRef } from 'react';
```

Comment:
Were any mods made? 

I'm wondering if I should scrutinize more closely.

Reply:
The only one is how we pass the options. 

I just noticed, I've not added test coverage for this hook, I'll move the tests over as well.